app/models/concerns/xml_message_object.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module XmlMessageObject
+  extend ActiveSupport::Concern
+
+  FORM_IDENTIFIER_PATTERN = /([^\/]+)\/(\d+\.\d+)\z/
+
+  included do
+    def pdf_transformation
```

Comment:
ak metoda nieco vyraba tak zvyknem pred nu davat `build_` alebo `prepare_` aby bolo jasne, ze to nieco robi a nie je to len getter.